### PR TITLE
Multiple tables

### PR DESCRIPTION
**Pull request by CarlosE-Feliz**
- Use primary key & foreign key mechanism for joining tables.✔
- Query multiple tables.✔
- Prepare complex queries that answer analytical questions.✔

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -29,44 +29,69 @@ INSERT INTO animals (name, date_of_brith, weight_kg, neutered, escape_attempts)
...
+
+INSERT INTO owners(id_owners, full_name, age)
+VALUES (6,'Jodie Whittaker', 38);
+
+
+
+INSERT INTO species (id_species, name)
+VALUES(1, 'Pokemon');
+INSERT INTO species (id_species, name)
+VALUES(2, 'Digimon');
```
Comment (oloomoses) [inline data.sql]: - [x] You have not implemented some of the requirements. Kindly take a look at the screenshot below and implement the points as highlighted in the project requirements. 
Screenshot from 2022-01-12 18-52-31

File: schema.sql [hunk 2]
```diff
@@ -1,11 +1,29 @@
...
+	full_name VARCHAR (30) NOT NULL,
+	age INT NOT NULL,
+    PRIMARY KEY (id_owners)
+)
+
+CREATE TABLE species (
+    id_species int NOT NULL,
+    name VARCHAR(30),
+    PRIMARY KEY (id_species)
+)
```
Comment (oloomoses) [inline schema.sql:30]: - [x] Kindly add queries to modify the `animals` table. The queries should: 
- Remove column `species`
- Add column `species_id` which is a foreign key referencing `species` table
- Add column `owner_id` which is a foreign key referencing the `owners` table

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hi @CarlosE-Feliz, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 2 above)
Comment (CarlosE-Feliz) [reply schema.sql:30]: Thanks for your review I really appreciate your effort on this  but I already have my foreign keys, I don't want to sound rude but the only thing that I'm missing there is to remove the column `species`

File: data.sql (hunk 1 above)
Comment (CarlosE-Feliz) [reply data.sql]: Thank you for noticing that! I totally forgot about that part! 😅😆😅

**Comment by CarlosE-Feliz** [comment]
## Note for the next CR 😋😋😋.

As I texted the previous CR through slack and get no answer, I just did the changes that I mention in the conversation.
I already have the foreign and primary keys for both tables, I just `DROP` the `species` column from animals and added the missing data.

Hoping now to get approved, cheers! 


# Schema.sql

- Removed column `species`
image

# Data.sql

- Missing data was added.

image

**Review by AshakaE — APPROVED** [review]
### Hi @CarlosE-Feliz   🙋‍♀️, 

# STATUS APPROVED ✔ 
   
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**